include/systems/variational_smoother_constraint.h

```diff
@@ -57,7 +57,7 @@ class PointConstraint
    * @param point The point defining the constraint.
    * @param tol The tolerance to use for numerical comparisons.
    */
-  PointConstraint(const Point &point, const Real &tol = TOLERANCE * TOLERANCE);
+  PointConstraint(const Point & point, const Real & tol = TOLERANCE);
```

Comment:
This is a pretty drastic loosening.  Six orders of magnitude with double precision!  What happened?